app/rest/runtime/src/main/java/io/syndesis/runtime/InfinispanCacheConfiguration.java

```diff
@@ -36,15 +37,15 @@
     private int maxEntries;
 
     @Bean
+    @ConditionalOnProperty(name = "JAVA_DEBUG", havingValue = "false", matchIfMissing = true)
```

Comment:
Couldn't find it in the docs, but does this fire also on environment variables or only on properties ?

Reply:
both, I think it goes to `Environment` that goes to `PropertySource`s, but that's from memory